js/src/modals/FirstRun/firstRun.js

```diff
@@ -130,13 +133,26 @@ export default class FirstRun extends Component {
         );
 
       case 2:
-        return (
+        const buttons = [
           <Button
             icon={ <ActionDone /> }
             label='Create'
+            key='create'
             disabled={ !canCreate }
-            onClick={ this.onCreate } />
-        );
+            onClick={ this.onCreate }
+          />
+        ];
+        if (hasAccounts) {
+          buttons.unshift(
+            <Button
```

Comment:
Would have preferred the labels/hint/text changes into `FormattedMessage` while we are in the component. (Saves work later, especially when adding new features). 

Not a crisis here though, so will just comment on it - small enough change overall.